### PR TITLE
Add option to auto-refresh logbook widget with latest update

### PR DESCRIPTION
**Pull request by algairim**
Signed-off-by: Mykola Mandra <mykola.mandra@cloudsoftcorp.com>

### CODE REVIEW

**Comment by algairim** [comment]
Proposed UI changes to support tail option, apart from query:

https://user-images.githubusercontent.com/81319331/125108898-8c85bf80-e0da-11eb-8ddd-1ea980de82da.mov

File: ui-modules/utils/logbook/logbook.js [hunk 1]
```diff
@@ -177,9 +177,27 @@ export function logbook() {
...
-                // TODO: implement logic for make output as table.
+                if ($scope.isLatest && $scope.logEntries.length !== 0) {
+                    if (logEntries.length > 0) {
+
+                        // Calculate date-time to display up to. Note, calendar does not take into account milliseconds,
+                        // round down to seconds.
+                        let latestDateTimeToDisplay = Math.floor(logEntries.slice(-1)[0].datetime / DEFAULT_NUMBER_OF_ITEMS) * DEFAULT_NUMBER_OF_ITEMS;
+
+                        // Display new log entries.
+                        let newLogEntries = logEntries.filter(entry => entry.datetime <= latestDateTimeToDisplay);
```
Comment (jathanasiou) [inline ui-modules/utils/logbook/logbook.js:188]: Would be a bit cleaner as
`const newEntries = logEntries.filter(({datetime})=> datetime <= latestDateTimeToDisplay);`

**Review by jathanasiou — APPROVED** [review]
Looks clean and to the point.

File: ui-modules/utils/logbook/logbook.js (hunk 1 above)
Comment (algairim) [reply ui-modules/utils/logbook/logbook.js:189]: Noted

**Review by jcabrerizo — APPROVED** [review]
LGTM @algairim, tks

**Pull request merged**